identify_and_start_isolating_side_effects.md

```diff
@@ -24,7 +24,7 @@ How to identify side-effects?
 - Can I precisely predict the result of executing some code?
 - If I were to execute the same code multiple times, would it always return the same values? (e.g. `sing_birthday_song('Alice', 44)` _vs_ `check_weather_outside()` or `roll_fancy_dice(12)`)
 
-If the answer to any of those questions is "no", then there is likely a side-effect somewhere in the function. This is not an exhaustive list, but that's the idea.
+If the answer to any of those questions is "no", then there is likely a side-effect somewhere in the function. This is not an exhaustive list, but it should give you a rough idea.
 
 How to start isolating side-effects?
```

Comment:
I had a look and both side-effects and side effects is grammatically correct. The latter is more commonly used, but I guess it is up to taste!

Reply:
I'm happy to move to the most widely used. Anything that looks more familiar removes unnecessary noise. :slightly_smiling_face: